lightning/src/ln/peer_handler.rs

```diff
@@ -864,7 +867,7 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 									peer.their_node_id = Some(their_node_id);
 									insert_node_id!();
 									let features = InitFeatures::known();
-									let resp = msgs::Init { features };
+									let resp = msgs::Init { features, net_address: None };
```

Comment:
Presumably this should be copied from the peer's `their_net_address` (and below)?